src/components/start-screen/StartScreen.tsx

```diff
@@ -1,9 +1,9 @@
 import React from "react";
-import { useMainStore } from "../hooks/useMainStore";
-import { GameStage } from "../stores/MainStore";
+import { useMainStore } from "../../hooks/useMainStore";
+import { GameStage } from "../../stores/MainStore";
 import styles from "./StartMenu.module.css";
 
-export const StartScreen = () => {
+export function StartScreen() {
```

Comment:
Any reason changing it to `export function`?

I think in the end you will benefit from components written like this `export const ComponentName: React.FC = () => {...}`

Reply:
But when it comes to non-React function like **utils** it is a good way to use regular **function** instead